### PR TITLE
small network patches

### PR DESCRIPTION
**Pull request by etbe**

### CODE REVIEW

File: policy/modules/admin/netutils.te [hunk 1]
```diff
@@ -142,6 +142,7 @@ logging_send_syslog_msg(ping_t)
 miscfiles_read_localization(ping_t)
 
 userdom_use_inherited_user_terminals(ping_t)
+term_use_unallocated_ttys(ping_t)
```
Comment (pebenito) [inline policy/modules/admin/netutils.te:145]: What is the use case for this?

File: policy/modules/services/networkmanager.fc [hunk 2]
```diff
@@ -15,7 +15,7 @@
 /etc/wicd/wireless-settings\.conf	--	gen_context(system_u:object_r:NetworkManager_etc_rw_t, s0)
 /etc/wicd/wired-settings\.conf	--	gen_context(system_u:object_r:NetworkManager_etc_rw_t, s0)
 
-/usr/lib/NetworkManager/nm-dispatcher.*	--	gen_context(system_u:object_r:NetworkManager_initrc_exec_t,s0)
+/usr/lib/NetworkManager/nm-dispatcher.*	--	gen_context(system_u:object_r:NetworkManager_exec_t,s0)
 /usr/lib/networkmanager/nm-dispatcher.*	--	gen_context(system_u:object_r:NetworkManager_initrc_exec_t,s0)
 /usr/libexec/nm-dispatcher.*	--	gen_context(system_u:object_r:NetworkManager_initrc_exec_t,s0)
```
Comment (pebenito) [inline policy/modules/services/networkmanager.fc:20]: This is labeling the same files but in different locations, so they should all be the same label.

That being said, please elaborate on why the current initrc labeling is incorrect.

File: policy/modules/system/libraries.if [hunk 3]
```diff
@@ -531,3 +531,21 @@ interface(`libs_relabel_shared_libs',`
...
+## <summary>
+##	watch lib dirs
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`libs_watch_shared_libs_dir',`
```
Comment (pebenito) [inline policy/modules/system/libraries.if:545]: Please make the "dir" plural, i.e. `libs_watch_shared_libs_dirs`

File: policy/modules/system/sysnetwork.if [hunk 4]
```diff
@@ -567,6 +567,24 @@ interface(`sysnet_manage_config',`
...
+## <summary>
+##     Watch a network config dir
+## </summary>
+## <param name="domain">
+##     <summary>
+##     Domain allowed access.
+##     </summary>
+## </param>
+#
+interface(`sysnet_watch_config_dir',`
```
Comment (pebenito) [inline policy/modules/system/sysnetwork.if:580]: Same plural "dirs" as above.

**Review by pebenito — CHANGES_REQUESTED** [review]

File: policy/modules/admin/netutils.te (hunk 1 above)
Comment (etbe) [reply policy/modules/admin/netutils.te:145]: Booting in recovery mode (boot failed enter root password) and running ping.

File: policy/modules/system/libraries.if (hunk 3 above)
Comment (etbe) [reply policy/modules/system/libraries.if:545]: done

File: policy/modules/system/sysnetwork.if (hunk 4 above)
Comment (etbe) [reply policy/modules/system/sysnetwork.if:580]: done

File: policy/modules/services/networkmanager.fc (hunk 2 above)
Comment (etbe) [reply policy/modules/services/networkmanager.fc]: I can't remember why I did it so I reverted that. I'll do more tests and submit at a future time with evidence if needed.

**Review by pebenito — APPROVED** [review]

**Pull request merged**